README.md

```diff
@@ -1,102 +1,96 @@
 # remult-better-auth
 
-Adapter to use [better-auth](https://www.better-auth.com) with [remult ORM](https://remult.dev)
+This adapter allows you to use [better-auth](https://www.better-auth.com) with [remult](https://remult.dev).
 
-## Installation
+## Default usage
 
-```console
-
-pnpm i @nerdfolio/remult-better-auth
+```sh
+# Install as dev dependencies.
+npm add better-auth @nerdfolio/remult-better-auth -D
 
+# run the generate command.
+npm run remult-better-auth generate
+# It will:
+#   - use `/src/modules/auth/server/better-auth-config.ts` or create it and use it. (You can escape this by passing `--config`).
+#   - generate entities `/src/modules/auth/entities.ts`. (You can escape this by passing `--output`).
 ```
 
-## Generate Schema
+## Examples
 
-### With @better-auth/cli
+### `better-auth-config.ts`
 
-Now that [better-auth PR3006](https://github.com/better-auth/better-auth/pull/3006) has been merged (v1.2.9 onward), you can use the @better-auth/cli as described by the better-auth team
+_usually in `src/modules/auth/server/better-auth-config.ts`_
```

Comment:
I don't think we should do this because it deviates away from the better-auth way of doing things and thus 1 more exception to the rule that I'll probably forget in the future. For better or for worse, better-auth's convention is a file named `auth.ts` in one of set of specific locations.

Reply:
okay 👌